decentraland/unity-renderer
# fix: Stop avatar movement at first tutorial step

## What does this PR change?

Fixes #2916 

A movement blocker has been added to avoid the movement and jumping of the avatar during the first step of the tutorial

## How to test the changes?

1. Clear the Decentraland's website data cache or launch the browser's incognito mode.
2. Launch the game as a guest.
3. Reach the Welcome screen section.
4. Use the movement keys while the Welcome screen is displayed.
5. Check that the avatar does not move when pressing movement keys (WASD or Jump).  

(Please see the attached video for reference: https://images.zenhubusercontent.com/337227404/493d22bb-6d9c-46c6-91f3-611c9481546a/_2916.mp4. In the video, you can see how the avatar is moved in that step. This should not happen)

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
